rero_ils/modules/api.py

```diff
@@ -137,7 +138,12 @@ def create(cls, data, id_=None, delete_pid=False,
         if not id_:
             id_ = uuid4()
         cls.minter(id_, data)
-        record = super(IlsRecord, cls).create(data=data, id_=id_, **kwargs)
+        # We don't need to keep the PID if any validation error
+        try:
+            record = super(IlsRecord, cls).create(data=data, id_=id_, **kwargs)
+        except (RecordValidationError, ValidationError) as e:
```

Comment:
One letter variables `e` are not recommended.

Reply:
I change it to `validation_error`